coded_tools/intranet_agents_with_tools/absence_manager.py

```diff
@@ -0,0 +1,264 @@
+import os
```

Comment:
Copied 1C tools here. @ofrancon is the absence_manager.py needed or can it be removed?

Reply:
What do you mean? `AbsenceManager` is used by `CheckLeaveBalancesTool` so it needed. It also shows an example of how to call an API.